packages/mds-trip-processor/src/proc-trip.ts

```diff
@@ -123,11 +121,11 @@ export async function tripProcessor() {
       )
 
       results.map(response => {
-        if (isUUID(response) && unprocessedTripsEvents[response]) delete unprocessedTripsEvents[response]
+        if (isUUID(response) && tripsEvents[response]) delete tripsEvents[response]
```

Comment:
Is there an alternate approach to using `delete`? Seems likely there is an approach that would use `reduce` instead.

Reply:
I need to think through this more in general, I'm afraid as is we'll run into race issues with how we're caching these trip events

Reply:
New PR will eliminate this logic